torchani/utils.py

```diff
@@ -192,7 +192,7 @@ def sae(self, species):
             intercept = self.self_energies[-1]
 
         self_energies = self.self_energies[species]
-        self_energies[species == torch.tensor(-1)] = torch.tensor(0)
+        self_energies[species == torch.tensor(-1, device=species.device)] = torch.tensor(0)
```

Comment:
how about the `torch.tensor(0)`?

Reply:
fixed in https://github.com/aiqm/torchani/pull/341/commits/9f3e6c2cf55ff4596c2e458712e399d0d9484e13